curriculum/challenges/english/08-coding-interview-prep/data-structures/create-a-map-data-structure.english.md

```diff
@@ -32,7 +32,7 @@ tests:
   - text: The Map data structure exists.
     testString: assert((function() { var test = false; if (typeof Map !== 'undefined') { test = new Map() }; return (typeof test == 'object')})(), 'The Map data structure exists.');
```

Comment:
```suggestion
    testString: assert((function() { var test = false; if (typeof Map !== 'undefined') { test = new Map() }; return (typeof test == 'object')})());
```

Reply:
This PR only removed assert message arguments which were not removed in the original two Coding Interview Prep PRs (#36410 and #36412).  If you look at those PRs, you will find this is removed on one of them.